client/src/pages/private/SiteTable.js

```diff
@@ -24,15 +25,37 @@ import { FeatureFlaggedComponent, flagKeys } from '../../services';
 import { fetchRegionSiteRows, fetchSiteRows } from '../../services/site';
 import { useTableStyles } from '../../components/tables/DataTable';
 
+const renderAllocations = (row) => {
+  const formattedDate = (date) => dayjs(row[date]).format('MMM D YYYY');
+  return (
+    <div>
```

Comment:
Can we use empty < Box >es instead of empty divs, just for consistency

Reply:
done!